### PR TITLE
[BB-1208] Ironwood preparation changes

### PR DESCRIPTION
**Pull request by lgp171188**
Configuration Pull Request
---
This PR ports the changes from the hawthorn branch which have not been upstreamed and have to be carried forward to ironwood. The PR is against the branch which will be made the `epodx` branch before the upgrade.

* Adds the `analytics_sandbox.yml` playbook and the nginx configuration template for the `analytics_api` app.

Additionally, it includes a change to optionally disable the demo role from being run.
Make sure that the following steps are done before merging

  - [ ] A devops team member has commented with :+1:
  - [ ] are you adding any new default values that need to be overridden when this goes live?
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] Are you adding/updating any variables that need to be added/updated to a specific `configuration-secure` repo?

### CODE REVIEW

File: playbooks/openedx_native.yml [hunk 1]
```diff
@@ -74,7 +74,8 @@
       nginx_sites:
       - edx_notes_api
       when: EDX_NOTES_API_ENABLED
-    - demo
+    - role: demo
+      when: COMMON_ENABLE_DEMO_ROLE
```
Comment (pomegranited) [inline playbooks/openedx_native.yml]: See [open-craft:configuration#96](https://github.com/open-craft/configuration/pull/96/files#diff-ae99e0327f2aceaa880f7b146a55b8faR37) -- this PR introduces `DEMO_ROLE_ENABLED: true`.  Ok to use this instead?

**Review by pomegranited — APPROVED** [review]
One change requested, but once that's resolved, this change looks good to me.
@lgp171188 

- [x] I didn't test this -- will verify functionality added after epodx deployment.
- [x] I read through the code and compared it to the [changes which haven't upstreamed for epod configuration](https://github.com/edx/configuration/compare/open-release/hawthorn.2...hks-epod:epodx).
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A

File: playbooks/openedx_native.yml (hunk 1 above)
Comment (lgp171188) [reply playbooks/openedx_native.yml]: I have left a comment about this [here](https://github.com/hks-epod/epodx/pull/21#issuecomment-492520225).

**Comment by lgp171188** [comment]
@pomegranited, I have updated the PR to rebase on top of the latest changes and removed the `COMMON_ENABLE_DEMO_ROLE` variable.

File: playbooks/openedx_native.yml (hunk 1 above)
Comment (pomegranited) [reply playbooks/openedx_native.yml]: 👍

**Comment by pomegranited** [comment]
@lgp171188 Thank you for addressing my [previous comment](https://github.com/open-craft/configuration/pull/97#pullrequestreview-237556073), this is good to merge!  👍

**Pull request merged**